.github/scripts/ci-test.sh

```diff
@@ -6,32 +6,32 @@ cd repos/jikesrvm
 # Test BaseBase builds
 # Only run one test
 
-# RBaseBaseNoGC
-python scripts/testMMTk.py -j $JAVA_HOME -g RBaseBaseNoGC -a "Xms1024M Xmx1024M" -- --use-third-party-heap=../../ --use-third-party-build-configs=../../jikesrvm/build/configs --use-external-source=../../jikesrvm/rvm/src
-# RBaseBaseSemiSpace
-python scripts/testMMTk.py -j $JAVA_HOME -g RBaseBaseSemiSpace -a "Xms75M Xmx75M" -- --use-third-party-heap=../../ --use-third-party-build-configs=../../jikesrvm/build/configs --use-external-source=../../jikesrvm/rvm/src
+# BaseBaseNoGC
```

Comment:
I think it is a good idea to differentiate the builds with Rust MMTk and the original MMTk. What is the reason for the renaming?

Also if we have to do the renaming, we need to update the scripts in ci-perf-kit: https://github.com/mmtk/ci-perf-kit/tree/master/scripts. I am not sure if there are other places where we use the config name, but we need to make sure this change does not break our CI checks.

Reply:
I think we have to do the renaming, because we plan to upstream the changes to JikesRVM, and the current upstream includes this renaming.
I will try to fix all occurances

Reply:
> because we plan to upstream the changes to JikesRVM, and the current upstream includes this renaming.

I don't understand this. We are going to upstream the support for a third party heap to JikesRVM. We are not going to upstream anything about our binding, and our configuration files are in our binding. Let me know if I misunderstood anything.

Reply:
No, you're right.
After double checking, we can still use any name for config files in the binding.

The main issue is that the default target config `x86_64-linux.properties` has changed to use the 64-bits layout (same as what was previously in `x86_64_m64-linux.properties`).
We either should revert the changes here, use `--m32` option with the `buildit` script, or use `--target-name=ia32-linux` with the `buildit` script.

The first approach doesn't work when we upstream.
The second and the third approaches will need updating all of the related scripts, as paths will e.g. change from `./dist/FastAdaptiveNoGC_x86_64-linux/rvm` to `./dist/FastAdaptiveNoGC_x86_64_m32-linux/rvm` or `./dist/FastAdaptiveNoGC_ia32-linux/rvm`.

Reply:
My point is that we would like to differentiate the builds from Rust MMTk - I think that is cleaner.

If the JikesRVM upstream has other changes that changes the build name and requires us to update our CI configs/scripts, we have to do it. You may need to do a search in ci-perf-kit for 'x86_64', I believe I have hard-coded it for JikesRVM in a few places. When you get this PR ready, you probably need to update the ci-perf-kit, and open a PR in mmtk-core to test the new ci-perf-kit with this PR.